pdf-converter/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const conveter = require('./pdf-to-image');

```

Comment:
typo in the variable name.
Is this supposed to be "converter"?

Reply:
We have removed this.